test/expect/TestJit.test_script_fibb.expect

```diff
@@ -0,0 +1,35 @@
+graph(%lim : UNKNOWN_TYPE) {
```

Comment:
Maybe we should just make the printer elide `: UNKNOWN_TYPE`. Originally I made it print this way to make it obvious when we were accidentally losing type information from a trace, but for these tests there's no point, without the shape analysis you're never going to have type info.